src/app/system/system-routing.module.ts

```diff
@@ -64,11 +68,41 @@ const routes: Routes = [
       },
       {
           path: 'data-tables',
-          component: ManageDataTablesComponent,
-          resolve: {
-                dataTables: ManageDataTablesResolver
-          },
           data: { title:  extract('Manage Data Tables'), breadcrumb: 'Manage Data Tables' },
+          children: [
+            {
+              path: '',
+              component: ManageDataTablesComponent,
+              resolve: {
+                dataTables: ManageDataTablesResolver
+              },
+            },
+            {
+              path: 'create',
+              component: CreateDataTableComponent,
+              data: { title: extract('Create Data Table'), breadcrumb: 'Create'},
+              resolve: {
+                codes: CodesResolver
+              }
+            },
+            {
+              path: 'view/:name',
```

Comment:
View and edit route paths need to be updated. Please see `accounting` section for reference.

Reply:
There is no numeric id of the data table, so using the name as id.

Reply:
That's okay still update is required with the way the route has been made and breadcrumb.

Reply:
So you want the route as `view/:name/edit`, right? 

Also, then I have to use `routeParamBreadcrumb`, correct?

Reply:
That is okay, use the name parameter but ensure consistency in the way url are being made.

Reply:
Check the latest update.